### ISSUE TITLE
Blocked by Haaretz since this morning

### ISSUE THREAD

**Issue by HarSuthep** [yuvadm/free-haaretz#32]
הדפדפן שברשותך מנסה לעקוף את מנגנון התשלום של הארץ
אתר זה זמין באופן מלא למנויים בלבד

**Comment by antishok** [comment]
Adding these filters to my adblocker (ublock origin) fixed it:
`||www.haaretz.co.il/htz/js/inter.js`
`||www.themarker.com/st/c/static/heb/inter.js`

(the themarker.com one is important for haaretz site too)

**Comment by HarSuthep** [comment]
thanks  !

**Comment by bokagit** [comment]
GREAT!

**Comment by Neter56** [comment]
Thanks, that works for me too!

**Comment by Nirsch1** [comment]
In ublock-origin, I've added the aforementioned 2 filters to the "My filters" tab.
But alas - I get a message that "the Haaretz site is blocked to surfers that use a pop-up blocker".
But when I put haaretz.co.il in the whitelist of ublock-origin, I get the "your blowser tries to bypass Haaretz's paywall" message all over again!
The whitelist is colliding with the filter-list!

**Comment by maxter161** [comment]
you should also add this to your filters:
||www.haaretz.co.il/htz/js/resp/minified/body-scripts-int.js

**Comment by LL25255252** [comment]
@maxter161 ,can you please post a link for example where this filter is needed (because after adding @antishok 2 filters everything seems to be OK for me).

**Comment by Nirsch1** [comment]
I'm using ublock-origin, and I've put the 3 filters in it. 
This add-on constantly tries to block the ads of Haaretz, which apparently "pisses them off".
So when I try to get, for instance, to https://www.haaretz.co.il/gallery/mejunderet/.premium-1.6516857 , 
I get redirected to https://www.haaretz.co.il/hblocked

**Comment by LL25255252** [comment]
@Nirsch1 no issues for me with the link you posted (uBO with @antishok 2 filters and without @maxter161 filter).

**Comment by Nirsch1** [comment]
@LL25255252 have you put Haaretz.co.il into the whitelist of ublock-origin?

**Comment by LL25255252** [comment]
> 
> 
> @LL25255252 have you put Haaretz.co.il into the whitelist of ublock-origin?

No (and I don't have any anti-adblock script/extension. Just uBO).

**Comment by antishok** [comment]
> 
> 
> ||www.haaretz.co.il/htz/js/resp/minified/body-scripts-int.js

I would __not__ recommend blocking that file - it contains many scripts that haaretz site uses. Blocking it causes comments on articles to not show up, and probably disables other functionality on the site too.

@Nirsch1 I don't know what to advise, except - make sure you enabled "EasyList Hebrew", delete all cookies/cache of haaretz.co.il

**Comment by antishok** [comment]
Another thing you can try: manually add a cookie `HtzPusr` with value 1.
You can do it like this:
- Browse to haaretz.co.il
- Open your browser's devtools JS console tab (F12 to open devtools)
- Run this line of code (paste it to the console and press enter):
      `document.cookie = 'HtzPusr=1'`

It might fool their code to make it think you're a paying customer. Seems to actually work, from my quick test

EDIT:  for themarker.com the cookie name is `TmPusr`

**Comment by Nirsch1** [comment]
> 
> 
> Another thing you can try: manually add a cookie `HtzPusr` with value 1.
> You can do it like this:
> 
>     * Browse to haaretz.co.il
> 
>     * Open your browser's devtools JS console tab (F12 to open devtools)
> 
>     * Run this line of code (paste it to the console and press enter):
>       `document.cookie = 'HtzPusr=1'`
> 
> 
> It might fool their code to make it think you're a paying customer. Seems to actually work, from my quick test

Ho my Gosh, @antishok, it actually works! 
uBlock feels like it filters the ads on Haaretz, but it actually doesn't!
Double cheers! :+1::+1:

**Comment by adibiton** [comment]
After adding this two lines to ublock origin filter
> ||www.haaretz.co.il/htz/js/inter.js
||www.themarker.com/st/c/static/heb/inter.js

This page works:
 https://www.haaretz.co.il/gallery/mejunderet/.premium-1.6516857

However this:
https://www.themarker.com/dynamo/cars/.premium-1.6529288
doesn't

Anyone has any clue?

**Comment by Nirsch1** [comment]
Have you followed @antishok's instructions?
i.e. https://github.com/yuvadm/free-haaretz/issues/32#issuecomment-426482052

**Comment by adibiton** [comment]
> Have you followed @antishok's instructions?
> i.e. [#32 (comment)](https://github.com/yuvadm/free-haaretz/issues/32#issuecomment-426482052)

Yes, is it working for you? (i'm getting 'for reading the full article buy...')

**Comment by LL25255252** [comment]
Everthing works fine for me with @antishok 's instructions. However there's an issue with Themarker comments.
No need for me to disable uBO on Haaretz but if I want to see the comments section on TheMarker I do need to disable uBO.
Tested on @adibiton link - https://www.themarker.com/dynamo/cars/.premium-1.6529288

**Comment by antishok** [comment]
@adibiton it's working for me.  have you installed the latest version of free-haaretz? previous versions used to only work on haaretz
You can try to do my steps from the comment above (adding the cookie manually) but on themarker.com instead of haaretz.co.il

@LL25255252 comments working fine for me on themarker, fwiw

**Comment by antishok** [comment]
Oh actually @adibiton, it's a different cookie name on themarker.com - cookie is `TmPusr`.
So on themarker.com you can try running: `document.cookie = 'TmPusr=1'`

**Comment by adibiton** [comment]
> @adibiton it's working for me. have you installed the latest version of free-haaretz? previous versions used to only work on haaretz
> You can try to do my steps from the comment above (adding the cookie manually) but on themarker.com instead of haaretz.co.il
> 
> @LL25255252 comments working fine for me on themarker, fwiw

After upgrading to the latest version - it's working!!

**Comment by LL25255252** [comment]
@adibiton can you see the comments on themarker page without disabling uBO (as I said before I can see them only if I disable UBO on themarker.com) ?

**Comment by adibiton** [comment]
@LL25255252 yes, i can see comments on themarker page without disabling uBO

**Comment by pashosh** [comment]
https://twitter.com/shefi89/status/1075386849009508352

**Comment by Nirsch1** [comment]
Hey, guys! A new blocking problem has recently appeared:

1. On Firefox (Windows 10, Windows 7 and Android), When using the "uBlock Origin" add-on, trying to open a paywalled article leads to a redirection to the page https://www.haaretz.co.il/hblocked .
2.  The adding of the following lines to "uBlock Origin" -> "My Filters" doesn't help:
    ||www.haaretz.co.il/htz/js/inter.js
    ||www.themarker.com/st/c/static/heb/inter.js
3.  The adding of the following cookies to F12 of Windows-Firefox doesn't work either:
    document.cookie = "HtzPusr=1"
    document.cookie = "TmPusr=1"

Any suggestions?

**Comment by Nirsch1** [comment]
I've added "EasyList Hebrew" to the filter-list of "uBlock Origin", and the problem seems to be solved! 
This can be done like so:
1. Go to https://filterlists.com
2. In the search bar enter: EasyList Hebrew.
3. In the "EasyList Hebrew" line, press on the button "Details".
4. Now press "Subscribe" and "OK".

That's it!
(Take a look at https://github.com/yuvadm/free-haaretz/issues/32#issuecomment-426475967 as well).
